core/services/chainlink/config_general_test.go

```diff
@@ -131,3 +126,142 @@ func TestConfig_LogSQL(t *testing.T) {
 	config.SetLogSQL(false)
 	assert.Equal(t, config.Database().LogSQL(), false)
 }
+
+//go:embed testdata/mergingsecretsdata/secrets-database.toml
+var databaseSecretsTOML string
+
+//go:embed testdata/mergingsecretsdata/secrets-explorer.toml
+var explorerSecretsTOML string
+
+//go:embed testdata/mergingsecretsdata/secrets-password.toml
+var passwordSecretsTOML string
+
+//go:embed testdata/mergingsecretsdata/secrets-pyroscope.toml
+var pyroscopeSecretsTOML string
+
+//go:embed testdata/mergingsecretsdata/secrets-prometheus.toml
+var prometheusSecretsTOML string
+
+//go:embed testdata/mergingsecretsdata/secrets-mercury_a.toml
+var mercurySecretsTOML_a string
```

Comment:
What's the difference between a and b? Is there a meaningful word that could be used instead?

And maybe try to avoid the underscore since it's not typically used in Go names

Reply:
Hmm, good point. The intent of splitting `mercurySecretsTOML` into two is so that I can test merging functionality for MercuryCredentials. Since it's a map, I'm testing that unique keys are successfully merged when split across config files. I could do something like `mercurySecretsTOMLSplitOne` and `mercurySecretsTOMLSplitTwo`? 

re: underscore, noted - thanks!